packages/shared/src/server/repositories/traces.ts

```diff
@@ -323,8 +325,8 @@ export const getTraceByIdOrThrow = async (
     FROM traces
     WHERE id = {traceId: String} 
     AND project_id = {projectId: String}
-    ${timestamp ? `AND timestamp = {timestamp: DateTime64(3)}` : ""} 
```

Comment:
@Steffen911 this is used in traces.byId and traces.byIdWithObservationsAndScores